libs/ledger-live-common/src/generated/bridge/js.ts

```diff
@@ -11,12 +11,15 @@ import filecoin from "../../families/filecoin/bridge/js";
 import hedera from "../../families/hedera/bridge/js";
 import near from "../../families/near/bridge/js";
 import neo from "../../families/neo/bridge/js";
-import polkadot from "../../families/polkadot/bridge/js";
 import ripple from "../../families/ripple/bridge/js";
 import solana from "../../families/solana/bridge/js";
 import stellar from "../../families/stellar/bridge/js";
 import tezos from "../../families/tezos/bridge/js";
 import tron from "../../families/tron/bridge/js";
+import { makeLRUCache } from "../../cache";
+import network from "../../network";
+import { withDevice } from "../../hw/deviceAccess";
+import { createBridge as polkadotCreateBridge } from "@ledgerhq/coin-polkadot/bridge/js";
```

Comment:
Sorry I may have forgotten if you told me already, but was this actually generated? As I don't see it in `libs/ledger-live-common/scripts/sync-families-dispatch.mjs`

Reply:
Yes, it is. You can take a look at line 108.